output/index.yaml

```diff
@@ -0,0 +1,110 @@
+  description: SwaggerHub API Auto Mocking
+paths:
+  /sentences:
+    get:
+      description: Returns a list of sentences
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Sentence'
+components:
+  schemas:
+    Sentence:
+      required:
+      - id
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the sentence
+          format: uuid
+        state:
+          pattern: "^\\d{2}"
+          type: string
+          description: State FIPS code
+        county:
+          pattern: "^\\d{5}"
+          type: string
+          description: County FIPS code
+        date:
+          type: string
+          description: "Sentencing date (month, day, and year)"
+          format: date
+        fine:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        restitution:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
+        confinement:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        suspension:
+          $ref: '#/components/schemas/SentenceType'
+        probation:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        community_service:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        treatment:
+          $ref: '#/components/schemas/SentenceTypeWithDuration'
+        death:
+          $ref: '#/components/schemas/SentenceType'
+        data_source:
+          $ref: '#/components/schemas/Sentence_data_source'
+    SentenceType:
+      type: object
+      properties:
+        hasType:
+          type: boolean
+    SentenceTypeWithDuration:
+      allOf:
+      - $ref: '#/components/schemas/SentenceType'
+      - type: object
+        properties:
+          minimum_duration:
+            type: integer
+          maximum_duration:
+            type: integer
+          duration_type:
+            type: string
+            enum:
+            - days
+            - months
+            - years
+    SentenceTypeWithFines:
+      allOf:
+      - $ref: '#/components/schemas/SentenceType'
+      - type: object
+        properties:
+          amount:
+            type: number
+    Sentence_data_source:
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the sentence in the source system
+        name:
+          type: string
+          description: Friendly name of the source system
+        type:
+          type: string
+          format: "enum [law_enforcement, courts, dept_of_corrections, criminal_history_repo,\
+            \ community_corrections_agency, other]"
+        raw_text:
+          type: string
+          description: Raw (unstructured) text string from the source system
+        confidence:
```

Comment:
Since we haven't picked any specific algorithms or services yet, it's impossible to know what the confidence output would look like. But 0-100 is pretty granular for downstream use and we might want to have "buckets" of confidence like low, medium, and high. We could store something more granular on our side so that it can be translated back and forth but I feel like that granularity might not be useful for SEARC.

Reply:
A fair point. Maybe confidence could be an object with score (0-100) and level (low, medium, high) properties, and score could be optional.